hal/src/msom/pinmap_defines.cpp

```diff
@@ -38,7 +38,7 @@ static hal_pin_info_t pinmap[TOTAL_PINS] = {
 /* D11           - 11 */ { RTL_PORT_A,     17, PIN_MODE_NONE, PF_NONE, ADC_CHANNEL_NONE, PWM_INSTANCE_NONE, PWM_CHANNEL_NONE, 0},
 /* D12           - 12 */ { RTL_PORT_A,     16, PIN_MODE_NONE, PF_NONE, ADC_CHANNEL_NONE, PWM_INSTANCE_NONE, PWM_CHANNEL_NONE, 0},
 /* D13           - 13 */ { RTL_PORT_A,     18, PIN_MODE_NONE, PF_NONE, ADC_CHANNEL_NONE, PWM_INSTANCE_NONE, PWM_CHANNEL_NONE, 0},
-/* D14 / A5      - 14 */ { RTL_PORT_B,      7, PIN_MODE_NONE, PF_NONE, 3,                1,                 17,               0},
+/* D14 / A5      - 14 */ { RTL_PORT_B,      3, PIN_MODE_NONE, PF_NONE, 6,                PWM_INSTANCE_NONE, PWM_CHANNEL_NONE, 0},
```

Comment:
We can handle the A5/A6 swap with the PMIC, but users that have EVT hardware and use these GPIO will need to ifdef their code to work on EVT and DVT hardware.